server/main.go

```diff
@@ -49,16 +50,28 @@ var (
 )
 
 func mlmdStoreOrDie() *mlmetadata.Store {
-	cfg := &mlpb.ConnectionConfig{
-		Config: &mlpb.ConnectionConfig_Mysql{
-			&mlpb.MySQLDatabaseConfig{
-				Host:     mySQLServiceHost,
-				Port:     proto.Uint32(uint32(*mySQLServicePort)),
-				Database: mlmdDBName,
-				User:     mySQLServiceUser,
-				Password: mySQLServicePassword,
+	var cfg *mlpb.ConnectionConfig
+	if *mlmdDBType == "sqlite" {
```

Comment:
This is not sqlite, this is the in-memory fake option. Change sqlite to `in-memory` instead? You may want to also provide the `sqlite` option as well. In which case, I would change the if else to use Go `switch` statement instead.

Reply:
Will do.

Reply:
Done.